writeengine/server/we_ddlcommandproc.cpp

```diff
@@ -916,13 +912,11 @@ uint8_t WE_DDLCommandProc::writeSyscolumn(ByteStream& bs, std::string& err)
 
             if (SCHEMA_COL == column.tableColName.column)
             {
-                boost::to_lower(schema);
                 colTuple.data = schema;
                 tmpStr = schema;
             }
             else if (TABLENAME_COL == column.tableColName.column)
             {
-                boost::to_lower(tablename);
```

Comment:
Do we not need the `if (lower_case_table_names)` blocks in this file?

Reply:
No -- because the global lower_case_table_names, which we pass to all those types of functions, is not available outside the plugin. WriteEngineServer doesn't have it. Changes in the plugin ensure that the proper case is set before being sent to the other processes